src/markdown-pages/automate-workflows/automated-tagging.mdx

```diff
@@ -14,103 +14,150 @@ resources:
-
-**Remember** Make sure to do the **Teardown** step at the end to decommision the entities that we provision in this guide. 
+In the next section, you'll learn about the tools you'll use throughout this guide to perform some of your tasks.
 
+</Intro>
 
 ## Before you begin
-To run this example there are several pre-requisite steps, most of which involve setting up the right configurations, and at least for now, building a [docker](https://docs.docker.com/get-docker/) image. 
 
-We use several of our open source projects to make this all work. Let's take a look at them now.
-* [Demo Deployer](https://github.com/newrelic/demo-deployer) - This is the engine that drives all the automation. For convenience, you'll clone this repo and build a docker image. In future iterations, we'll publish the image in docker hub or someplace similar. 
-* [Demo Nodetron](https://github.com/newrelic/demo-nodetron) - This is the simple app that will be provisioned on a host. It will give you a simple web page and an API that you can hit with a tool like [Postman](https://www.postman.com/downloads/) depending on how much traffic you want to generate. 
-* [Demo New Relic Instrumentation](https://github.com/newrelic/demo-newrelic-instrumentation) - This is the project that we use to layer in the instrumentation for the infrastructure and the application. 
+Throughout this guide, you'll use three open source projects to automatically set up, instrument, and teardown your infrastructure:
+
+- [Demo Deployer](https://github.com/newrelic/demo-deployer): The engine that drives all the automation
+- [Demo Nodetron](https://github.com/newrelic/demo-nodetron): A Node.js app that you will provision on your host. It will present a web page and an API, which you will use to generate traffic.
+- [Demo New Relic Instrumentation](https://github.com/newrelic/demo-newrelic-instrumentation): A project that you will use to instrument your host and application
+
+Each project is distinct, and encapsulates its own behavior. This maintains a separation of concerns and allows you to compose complex scenarios using a modular approach.
+
+<Tip title="Collaborate with us">
+
+The Demo Deployer is currently in development. Because of our commitment to open source, we are working in the open early in the process and invite you to collaborate with us. Drop any thoughts and comments in the [Build on New Relic](https://discuss.newrelic.com/c/build-on-new-relic) support thread, and let us know what you think!
+
+</Tip>
 
-We kept each project distinct to encapsulate its behavior. The idea being that we can simply add more behaviors and then weave them in to assemble more complex scenarios. Cool, right?
+Now that you have a high-level understanding of the tools you will use, you can begin by building the Demo Deployer!
```

Comment:
bring on the contractions! I can see keeping the future tense here. And recommend contractions 'the tools you'll use."